### ISSUE TITLE
Fix contact info in cabal file

### ISSUE THREAD

**Issue by hce** [IreneKnapp/hs-captcha#2]
The e-mails bounce, the website doesn't work anymore. I'd like to suggest to change the URL to the github one.

**Comment by IreneKnapp** [comment]
Agreed. For my information wrt prioritization, are you actually using this library?

**Comment by hce** [comment]
Yup. If you like, I can send you a link to the site where I’m using it once it is online, which should be in ~1 month.

**Comment by IreneKnapp** [comment]
Yeah, I'd love that! You can see in the commit history that this code was all written in one attempt, nine years ago, so I'm quite flattered that you find it useful. I'll get the contact info cleaned up for you. :)

**Comment by hce** [comment]
As promised: https://cloud.virtualforge.com/v1/#/resetpw  :-)
